src/stories/ConfirmButton.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import ConfirmButton, { ConfirmButtonProps } from '#components/ConfirmButton';
+
+export default {
+    title: 'Action/ConfirmButton',
+    component: ConfirmButton,
+    argTypes: {},
+};
+
+const ConfirmationHeader = (
+    <h2>Confirm Action?</h2>
+);
+
+const Template = (args: ConfirmButtonProps) => (
+    <ConfirmButton
+        {...args}
+        name="confirm-button"
+        confirmationHeader={ConfirmationHeader}
+        confirmationMessage="Confirmation message"
+        onConfirm={() => console.log('confirmed')}
```

Comment:
You don't need to add handler on stories. Storybook automatically injects it for you!